### PR TITLE
Translation in french

### PR DESCRIPTION
**Pull request by WhitePomme**
I translated in french (in fr.md files) two file into the folders:
- "osu!wiki_contribute"
- and "Do_you_really_[...]_peppy"

And i have rectify some errors in the page "en.md" in "osu!wiki_contribute".

### CODE REVIEW

**Comment by RaikireHiuduo** [comment]
See #123 for osu!wiki_contribute.

The errors you fixed are negligible in en.md for osu!wiki_contribute.

**Comment by WhitePomme** [comment]
Ah ok i understand !

File: wiki/Do_you_really_want_to_ask_peppy/fr.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Salut, je comprends votre urgence de me contacter personnellement, mais je vous devez comprendre que ce n'est **pas** un endroit pour me questionner pour de l'aide. Je garde osu! support separé de mes messages personnelles, et cela est déjà un travail assez long. Je vous demande donc de suivre une de ces options.
```
Comment (MegaApplePi) [inline wiki/Do_you_really_want_to_ask_peppy/fr.md]: Add the title to the top of this page (above this line) The English title is `# Do you really want to as peppy?`

**Comment by sr229** [comment]
Update this to the new version then do another re-translation.

**Comment by Repflez** [comment]
Closing due to fork being deleted on OP's side.

**Pull request closed without merging**